discovery-provider/src/tasks/entity_manager/entity_manager.py

```diff
@@ -271,6 +280,19 @@ def entity_manager_update(
                 except IndexingValidationError as e:
                     # swallow exception to keep indexing
                     logger.info(f"failed to process transaction error {e}")
+                except Exception as e:
+                    indexing_error = IndexingError(
+                        "tx-failure",
+                        block_number,
+                        hex_blockhash,
+                        txhash,
+                        str(e),
+                    )
+                    skipped_tx_hash = create_and_raise_indexing_error(
+                        indexing_error, update_task.redis, session
+                    )
+                    logger.info(f"skipping transaction hash {skipped_tx_hash}")
```

Comment:
do we need to log here if we log the tx hash inside create_and_raise_indexing_error? just the naming seems confusing the fn is called `create_and_raise_indexing_error` but also returns a skipped tx hash. if we wanna keep the log here maybe rename the fn?

Reply:
yea true and the tx hash is already in the structured logger context, gonna remove this